### ISSUE TITLE
Continuous Learning in a Hierarchical Multiscale Neural Network

### ISSUE THREAD

**Issue by ymym3412** [ymym3412/acl-papers#147]
## 0. 論文
[Continuous Learning in a Hierarchical Multiscale Neural Network](https://arxiv.org/abs/1805.05758)
Thomas Wolf, Julien Chaumond, Clement Delangue

## 1. どんなもの？
短期・中期・長期の関係性を扱う階層型のニューラル言語モデル

## 2. 先行研究と比べてどこがすごい？
短期・中期・長期の関係性を明示的に扱う

## 3. 技術や手法のキモはどこ？
系列の中の短期(Short-term)、中期(Medium-term)、長期(Long-term)を関係性を扱うMemroyを持った階層構造のニューラル言語モデル
低レイヤのパラメータは上位のレイヤから与えられる。上位のレイヤが低レイヤのmeta-learnerとして機能する

![image](https://user-images.githubusercontent.com/9605058/40368270-5542157c-5e16-11e8-9d55-b632708c39e9.png)

## 4. どうやって有効だと検証した？
Wikitext-2の言語モデルタスクでperplexityで計測し46.9を記録。
Medium-termのレイヤを加えたときが改善幅が大きい

## 5. 議論はある？
Medium-termのレイヤをFFとRNNで比較するとRNNではそれほどうまくいかなかった
meta-learnerの役割として情報をストアしておくことは重要ではなかったのではとしている

## 6. 次に読むべき論文は？
[Learning to learn by gradient descent by gradient descent](https://arxiv.org/abs/1606.04474)
Marcin Andrychowicz, Misha Denil, Sergio Gomez, Matthew W. Hoffman, David Pfau, Tom Schaul, Brendan Shillingford, Nando de Freitas

[Nested LSTMs](https://arxiv.org/abs/1801.10308)
Joel Ruben Antony Moniz, David Krueger
